src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -130,13 +130,13 @@ export class RbdFormComponent extends CdForm implements OnInit {
     this.resource = $localize`RBD`;
     this.features = {
       'deep-flatten': {
-        desc: $localize`Deep flatten`,
+        desc: $localize`Deep flatten (feature can be disabled but can't be re-enabled later)`,
```

Comment:
I see that there is a [toggle to enable](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html#L235) the `cd-helper` component already present in the form.  Its just that the `helperHtml` is missing in the [RbdImageFeature](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-feature.interface.ts) interface. So you can add an optional `helperHtml` key there and move the helper texts into that item so that it'll be shown only in a helper.

Reply:
So if I understand correctly you're suggesting to add `helperHtml` key in image feature interface?

Reply:
Yup. On optional key.

Reply:
Done!